scripts/mkcloud

```diff
@@ -1113,6 +1175,14 @@ Optional
         Defines which scenario file should be used. Only works with cloud5 or newer.
         Currently only the step 'batch' uses such a file.
         Scenario files have to placed in \${mkcloud_dir}/scenarios/cloud\$(getcloudver).
+
+Crowbar Development Options:
+    crowbar_dev=1 (default='')
+        Install dependencies for a Crowbar Development app on the Admin node
```

Comment:
... or use this switch as switch to do all of:
- install deps
- source fancy lib
- change step list (if there's only one way to get a crowbar-dev env)